examples/lazy-injection/counter-todo/src/features/todos/Todos.tsx

```diff
@@ -0,0 +1,19 @@
+import { useAppSelector } from "../../app/hooks"
+import { AddTodo } from "./AddTodo"
+import { Todo } from "./Todo"
+import { selectTodoIds } from "./todoSlice"
+import styles from "./Todos.module.css"
+
+export function Todos() {
+  const todoIds = useAppSelector(selectTodoIds)
+  return (
+    <div className={styles.todos}>
+      <div className={styles.todosList}>
+        {todoIds.map(id => (
+          <Todo key={id} id={id} />
+        ))}
+      </div>
+      <AddTodo />
```

Comment:
```suggestion
            <AddTodo />
      <div className={styles.todosList}>
        {todoIds.map(id => (
          <Todo key={id} id={id} />
        ))}
      </div>
```

Looks like currently the todos get added above the input field

Reply:
yeah, that was just how i did it - i guess it makes sense the other way though